### PR TITLE
tests(dlp): normalize VPCSC configuration in systests

### PR DESCRIPTION
**Pull request by tseaver**
Toward #9580

### CODE REVIEW

File: dlp/tests/system/gapic/v2/test_system_dlp_service_v2_vpcsc.py [hunk 1]
```diff
@@ -42,480 +36,306 @@ def _is_rejected(call):
...
-        PROJECT_OUTSIDE is None,
-        reason="Missing environment variable: GOOGLE_CLOUD_TESTS_VPCSC_OUTSIDE_PERIMETER_PROJECT",
-    )
+            assert not (self._is_rejected(delayed_outside))
+            assert self._is_rejected(delayed_inside)
+
+    @vpcsc_config.skip_unless_inside_vpcsc
+    @vpcsc_config.skip_unless_outside_project
     def test_inspect_content(self):
+        import pdb; pdb.set_trace()
```
Comment (busunkim96) [inline dlp/tests/system/gapic/v2/test_system_dlp_service_v2_vpcsc.py]:
```suggestion
```

Comment (tseaver) [reply dlp/tests/system/gapic/v2/test_system_dlp_service_v2_vpcsc.py]: D'oh! 0584705

**Comment by tseaver** [comment]
I tagged with "do not merge" because I believe we should refactor all these un-idiomatic tests along the lines of the changes in #9618.

**Comment by tseaver** [comment]
@busunkim96 I removed the "do not merge" label after updating the tests to follow the pattern in #9618.

**Pull request merged**